test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -12639,7 +12639,26 @@ def isUp(self):
         """
         Check for PBS mom up
         """
-        return super(MoM, self)._isUp(self)
+        # Poll for few seconds so we don't miss any signals sent to
+        # mom
+        for _ in range(5):
```

Comment:
The server's isUp() will check once every second for 20 seconds.  Should we do the same here?  The server's use case is different, so I'm not sure.

Reply:
poll for 20s sounds good

Reply:
server’s isUp() 20 sec polling is little different than this.  When server is up, polling stops and isUp() returns. But here we would be doing a 20 sec polling unconditionally (always).  Wouldn’t 20 seconds be very long in a ‘mom and nodes already up and no signal sent to mom’ case ? 
In my tests on few test machines , 3-5 seconds polling was sufficient to catch any signal sent to Mom.  I am not sure if that would be sufficient for all machines though. Any thoughts ?